addons/analytic/models/analytic_plan.py

```diff
@@ -22,7 +22,10 @@ def _default_color(self):
         string="Parent",
         ondelete='cascade',
         check_company=True,
-        domain="[('id', '!=', id)]",
+        domain="""[
+            ('id', '!=', id),
+            '!', ('id', 'child_of', id)
+        ]""",
```

Comment:
It's small enough to be on one line now :smile:

Reply:
Size does not matter :o